src/Extension/CoreExtension.php

```diff
@@ -1144,9 +1144,13 @@ public static function nl2br($string): string
      * @param string|null $content
      *
      * @internal
+     *
+     * @deprecated since Twig 3.12
      */
     public static function spaceless($content): string
     {
+        trigger_deprecation('twig/twig', '3.12', 'The "spaceless" filter is deprecated.');
```

Comment:
Do we need a runtime deprecation for each usage of the filter ? can't we rely only on the compile-time deprecation instead ?

Reply:
OK, actually, the compile-time deprecation is not implemented.